src/main/java/com/gmail/yuramitryahin/service/UserMapperImpl.java

```diff
@@ -0,0 +1,13 @@
+package com.gmail.yuramitryahin.service;
+
+import com.gmail.yuramitryahin.model.User;
+import com.gmail.yuramitryahin.model.dto.UserResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapperImpl implements UserMapper {
+    public UserResponseDto toDto(User user) {
+        return new UserResponseDto(user.getFirstName(), user.getFirstName(),
```

Comment:
```suggestion
        return new UserResponseDto(user.getFirstName(), user.getLastName(),
```

Reply:
this constructor is getting really big. Let's use setters

Reply:
fixed